apps/vr-tests-react-components/src/stories/AvatarGroup.stories.tsx

```diff
@@ -38,14 +44,23 @@ const names = [
 
 const sizes = [16, 20, 24, 28, 32, 36, 40, 48, 56, 64, 72, 96, 120, 128];
 
-const AvatarGroupList: React.FC<AvatarGroupProps> = props => {
+const inlineAvatars = 4;
+
+const AvatarGroupList: React.FC<
+  AvatarGroupProps & { overflowIndicator?: AvatarGroupOverflowProps['overflowIndicator'] }
+> = props => {
   return (
-    <div style={{ display: 'flex', flexDirection: 'column', flexWrap: 'wrap', gap: '10px' }}>
+    <div style={{ display: 'flex', flexDirection: 'column', flexWrap: 'wrap', gap: '10px', padding: '10px' }}>
       {sizes.map(size => (
         <AvatarGroup key={size} size={size as AvatarGroupProps['size']} {...props}>
-          {names.map(name => (
-            <AvatarGroupItem key={name} name={name} />
+          {names.slice(-inlineAvatars).map(name => (
+            <AvatarGroupItem key={'inline' + name} name={name} />
           ))}
+          <AvatarGroupOverflow overflowIndicator={props.overflowIndicator}>
+            {names.slice(0, -inlineAvatars).map(name => (
+              <AvatarGroupItem key={name} name={name} />
+            ))}
+          </AvatarGroupOverflow>
         </AvatarGroup>
```

Comment:
Like we talked about offline, it would be nice to have a utility function to do this array slicing, since this will likely be a common way to use AvatarGroup.